components/SubTrackDetails/SubTrackDetails.jsx

```diff
@@ -0,0 +1,40 @@
+require('./SubTrackDetails.scss')
+
+import React from 'react'
+import classNames from 'classnames'
+
+const SubTrackDetails = ({name,code,description,tracks}) => {
+  const developClass = classNames(
+    {'verticalLine develop': true},
+    {hidden: tracks.indexOf('develop') === -1 }
+  )
```

Comment:
Better syntax to use would be `classNames('verticalLine', 'develop', {hidden: tracks.indexOf('develop') === -1})`